src/lib/frontend/endpoints/Endpoints.ts

```diff
@@ -10,27 +8,29 @@ import type {OrderDetailsPresentation} from '$lib/frontend/presentations/OrderDe
 import {API_BASE_ENDPOINT, BASE_HEADERS} from '$lib/frontend/Constants'
 import type {BusinessContactRequest} from '$lib/frontend/requests/BusinessContactRequest'
 import type {GenericSuccessResponsePresentation} from '$lib/frontend/presentations/GenericSuccessResponsePresentation'
+import type {CheckoutRequest, CheckoutResponse} from '$lib/frontend/requests/CheckoutRequest'
+import {EndpointError} from '$lib/frontend/endpoints/EndpointError'
 
-export async function createCheckoutSession(parameters: CheckoutRequest): Promise<CheckoutResponse | undefined> {
+export async function createCheckoutSession(parameters: CheckoutRequest): Promise<CheckoutResponse> {
     const response = await fetch(
         `${API_BASE_ENDPOINT}/cart/checkout`,
         {
             method: 'POST',
             headers: BASE_HEADERS,
-            body: JSON.stringify(parameters),
+            body: JSON.stringify(parameters)
```

Comment:
allowed trailing comma should never be removed because they help prevent merge conflicts